1-exercises/A-array-find/exercise.js

```diff
@@ -16,8 +16,11 @@ let names = [
   "Karim",
   "Ahmed",
 ];
+ //function longNameWithA(name) {
+ // return name.length > 7;
+//}
 
-let longNameThatStartsWithA = findLongNameThatStartsWithA(names);
```

Comment:
For this exercise you should define your own function `findLongNameThatStartsWithA` and implement your logic there.